release.sh

```diff
@@ -97,6 +97,18 @@ docker build --no-cache --build-arg version=${VERSION} --tag gisaia/arlas-wui:${
 docker push gisaia/arlas-wui:${VERSION}
 docker push gisaia/arlas-wui:latest
 
+echo "==> Npm"
+rm -rf dist
+npm install
+npm run ngc-build
```

Comment:
I think the `ngc-build` script is missing in package.json

Reply:
are you sure about that ?
https://github.com/gisaia/ARLAS-wui/blob/3c9f112c3fcc3754dc0e9abbc82de24c5cad2ee9/package.json#L10